viz_scripts/scaffolding.py

```diff
@@ -1,7 +1,9 @@
 import pandas as pd
 import numpy as np
+import matplotlib.pyplot as plt
 import sys
 from collections import defaultdict
+from collections import OrderedDict
```

Comment:
cleanup fix: I will accept this since we already use `defaultdict`, but please change these to use the`import X from Y` coding standard that we use otherwise